pkg/api/v1/types.go

```diff
@@ -394,3 +395,47 @@ func (ml *MigrationList) UnmarshalJSON(data []byte) error {
 	*ml = tmp2
 	return nil
 }
+
+// Given a pod, create a NodeSelectorTerm with anti-affinity for that pod's node.
+// This is useful for the case when a migration away from a node must occur.
+// In this context, pod is where a migration is starting
+func AntiAffinityFromNode(pod *v1.Pod) *v1.NodeSelectorTerm {
```

Comment:
We need to take the node from the VM, since we don't have the pod of the VM where we are calling this.